cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -276,17 +276,67 @@ TEST(RayClusterModeTest, GetActorTest) {
   EXPECT_FALSE(ray::GetActor<Counter>("not_exist_actor"));
 }
 
-TEST(RayClusterModeTest, CreateAndRemovePlacementGroup) {
+ray::PlacementGroup CreateSimplePlacementGroup(const std::string &name) {
   std::vector<std::unordered_map<std::string, double>> bundles{{{"CPU", 1}}};
 
+  ray::internal::PlacementGroupCreationOptions options{
+      false, name, bundles, ray::internal::PlacementStrategy::PACK};
+  return ray::CreatePlacementGroup(options);
+}
+
+TEST(RayClusterModeTest, CreateAndRemovePlacementGroup) {
+  auto first_placement_group = CreateSimplePlacementGroup("first_placement_group");
+  EXPECT_THROW(CreateSimplePlacementGroup("first_placement_group"),
+               ray::internal::RayException);
+  ray::RemovePlacementGroup(first_placement_group.GetID());
+}
+
+TEST(RayClusterModeTest, CreatePlacementGroupExceedsClusterResource) {
+  std::vector<std::unordered_map<std::string, double>> bundles{{{"CPU", 100}}};
```

Comment:
Maybe we should give a bigger cpu number? I concern this test run in a big machine.